processor/README.md

```diff
@@ -133,15 +133,29 @@ Use the token to authenticate requests protected by JWT: `Authorization: Bearer
 The processor exposes following endpoints to execute various operations with Adyen platform:
 
 ### Get payment methods
-TBC
+It obtains the available payment methods supported by the given Adyen account.
+#### Endpoint
+`POST /payment-methods`
+
+#### Request Parameters
+The request body is same as [adyen checkout payment methods request](https://docs.adyen.com/api-explorer/Checkout/71/post/paymentMethods#request) except following parameters are not required
+- amount
+- merchantAccount
+- countryCode
+
+#### Response Parameters
+- paymentMethods: Detailed list of payment methods required to generate payment forms.
+- storedPaymentMethods : List of all stored payment methods.
```

Comment:
this would be something similar. it is prepared for the future, in order to accept the stored payment methods but it's not possible with the current version so maybe it's better to not mention it as it could be confusing

Reply:
https://github.com/commercetools/connect-payment-integration-adyen/pull/104/commits/5299dd2354b51e7b3167a1d3b70e5670df77803a